fastify.js

```diff
@@ -708,7 +708,7 @@ function fastify (options) {
           const reply = new Reply(res, request, childLogger)
           return frameworkErrors(new FST_ERR_ASYNC_CONSTRAINT(), request, reply)
         }
-        const body = '{"error":"Internal Server Error","code":"FST_ERR_ASYNC_CONSTRAINT","message":"Unexpected error from async constraint","statusCode":500}'
```

Comment:
Can we revert this change?
The change will need further discussion 🙂

Reply:
@metcoder95 Certainly. Reverted (i.e. the `code` prop removed), commit https://github.com/fastify/fastify/pull/4632/commits/d1dc335c7048620caa8267d74562141b67e0b09c